libraries/app/application.cpp

```diff
@@ -1047,6 +979,10 @@ void application::set_program_options(boost::program_options::options_descriptio
           "For database_api_impl::get_settle_orders and get_settle_orders_by_account to set its default limit values as 300")
          ("api-limit-get-assets",boost::program_options::value<uint64_t>()->default_value(101),
           "For database_api_impl::list_assets and get_assets_by_issuer to set its default limit values as 101")
+         ("accept-incoming-connections", bpo::value<bool>()->implicit_value(true), "Accept incoming connections")
+         ("advertise-peer-algorithm", bpo::value<string>()->implicit_value("all"), "Determines which peers are advertised")
+         ("advertise-peer-list", bpo::value<vector<string>>()->composing(), 
+            "P2P nodes to advertise (may specify multiple times")
```

Comment:
...advertise or exclude, depending on advertise-peer-algorithm...

Reply:
IMHO better have different list parameters for excluding and including. When copying config files among nodes, sharing one param would lead to unexpected results if not careful enough.

Reply:
Agreed. Will add a separate parameter